config.js

```diff
@@ -0,0 +1,3 @@
+const dev = process.env.NODE_ENV !== "production";
+
+export const server = dev ? "http://localhost:3000" : "https://wheretomug.ml";
```

Comment:
```ts
export const server = dev ? "http://localhost:3000" : "https://wheretomug.ml";
```

I suspect it has got to do with this server variable

https://vercel.com/docs/environment-variables#system-environment-variables

Reply:
The preview is still not working. I think it is cause of the preview server url "https://where-to-mug-git-refactor-extractfiltertoapi-okkar.vercel.app". 
I have set the server url in production to be the normal url "https://wheretomug.ml".
![Screenshot_20210702-143233_Chrome](https://user-images.githubusercontent.com/70012669/124231598-0f3ed580-db43-11eb-893d-2d5bee67146d.jpg)

Reply:
Could you check it you are able to utilize the built-in env vars?

https://vercel.com/docs/environment-variables#system-environment-variables

Reply:
Updated and committed change.

Reply:
The idea is to dynamically get the deployment URL if it's not local dev